docs/contributing.md

```diff
@@ -0,0 +1,160 @@
+
+  ```
+  $ git push origin { YOUR_BRANCH_NAME }
+  ```
+
+7. In Github, navigate to [carbon-design-system/carbon-addons-website](https://github.com/carbon-design-system/carbon-addons-website) and click the button that reads "Compare & pull request".
+
+8. Write a title and description, the click "Create pull request".
+
+  See [how to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request) for more details on writing good PRs.
+
+9. Stay up to date with the activity in your pull request. Maintainers will be reviewing your work and making comments, asking questions and suggesting changes to be made before they merge your code. When you need to make a change, add, commit and push to your branch normally.
+
+  Once all revisions to your pull request are complete, a maintainer will squash and merge your commits for you.
+
+
+**That's it! Thank you for your contribution!**
+
+
+## Coding Standards
+
+To ensure consistency throughout the source code, keep these rules in mind as you are working:
+
+### Style Guide
+
+For a set of basic rules and guidelines for developing React components, see [here](https://github.com/airbnb/javascript/tree/master/react#basic-rules).
+
+Feel free to edit/write components in your own style but be wary that we may ask you to make changes while reviewing your pull request.
+
+### Linting
+
+We enforce some style rules for code in this repository using [eslint](http://eslint.org/). You can install a linting addon to a lot of editors and IDEs that will follow our linting rules.
+
+If you decide to not install a linter addon, or cannot, you can run `npm run lint` to get a report of any style issues. Any issues not fixed will be caught during CI, and will prevent merging.
+
+## Commit Message Guidelines
+
+We use commit message guidelines based on the [Angular Commit Conventions](https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit), using `Commitizen` as a CLI wizard to walk developers through writing their commit message.
+
+This CLI wizard can either be installed by the developer to the global `npm` scope by running `npm i -g commitizen`, or it can be used as a dev dependency on this project by running `npm run commit`.
+
+After the commit message has been submitted, it is checked by [`husky`](https://www.npmjs.com/package/husky) and [`commitlint`](https://www.npmjs.com/package/commitlint) to ensure it is syntactically correct.
+
+## Testing
+
+If you add any features to our code, make sure to add tests so that your changes are covered. Tests are written using [JEST](https://github.com/facebook/jest). You can see how well your code is covered by looking at the `.gh-pages/coverage/lcov-report/index.html` file after running the coverage command.
+
+Test your changes by running our test commands:
+
+* Run linting:
+
+  ```
+  npm run lint
```

Comment:
Should we update these commands to use `yarn` since that's what we prefer?

Reply:
OK good to articulate our `yarn` preference - Will do